### PR TITLE
Syntactic sugar for Event<T>

### LINKED ISSUE #1643: Add syntactic sugar for Event constructors
All the boxing is really terrible, and will hopefully be able to go away in a future Swift release. Consequently, we should have custom constructor functions that don't require an explicit `Box`, like LlamaKit's `success()` and `failure()` functions.

### PR DESCRIPTION
**Pull request by jspahrsummers**
Resolves #1643.

### CODE REVIEW

File: ReactiveCocoa/Swift/ColdSignal.swift [hunk 1]
```diff
@@ -1359,6 +1381,21 @@ extension Event: Printable {
 	}
 }
 
+/// Puts a `Next` event into the given sink.
+public func sendNext<T>(sink: SinkOf<Event<T>>, value: T) {
```
Comment (joshaber) [inline ReactiveCocoa/Swift/ColdSignal.swift]: Is there a reason not to make these funcs on `SinkOf`?

**Comment by joshaber** [comment]
:confetti_ball:

File: ReactiveCocoa/Swift/ColdSignal.swift (hunk 1 above)
Comment (jspahrsummers) [inline ReactiveCocoa/Swift/ColdSignal.swift]: They're restricted in the type of `SinkOf` accepted.

**Comment by joshaber** [comment]
:sunglasses:

**Pull request merged**